cli/tools/coverage/style.css

```diff
@@ -43,8 +41,6 @@ pre {
     monospace;
   margin: 0;
   padding: 0;
-  -moz-tab-size: 2;
-  -o-tab-size: 2;
```

Comment:
`tab-size` is sufficient and well-supported: https://developer.mozilla.org/en-US/docs/Web/CSS/tab-size#browser_compatibility